modules/ebi-metagenomics/deeptmhmm/meta.yml

```diff
@@ -0,0 +1,45 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: deeptmhmm
+description: A Deep Learning Model for Transmembrane Topology Prediction and Classification
+keywords:
+  - sort
+  - example
+  - genomics
+tools:
+  - deeptmhmm:
+      description: Deep Learning model for Transmembrane Helices protein domain prediction through the BioLib Python Client
+      homepage: https://dtu.biolib.com/DeepTMHMM
+      documentation: https://dtu.biolib.com/DeepTMHMM
+      doi: 10.1101/2022.04.08.487609
+      licence: ["MIT"]
+
+input:
+  - fasta:
+      type: file
+      description: Database of sequences in FASTA format
+      pattern: "*.{fasta,fa,fasta.gz,fa.gz}"
+
+output:
+  - gff3:
+      type: file
+      description: Predicted topologies in general Feature Format Version 3
+      pattern: "biolib_results/TMRs.gff3"
```

Comment:
Is this correct? According to the docs it should be:

- input/output patterns (if present) MUST follow a [Java glob pattern](https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob)

Reply:
should only be for the file? and not the path?

Reply:
Nah this is more like it. I updated it with some more info and also added the optional outputs I had forgotten.